src/new-components/columns/columns.module.css

```diff
@@ -0,0 +1,142 @@
+/* Default horizontal spacing */
+.space-xsmall {
+    margin-left: calc(var(--reactist-spacing-xsmall) * -1);
```

Comment:
Will we go back to .less here to keep IE11 support for now?

Reply:
I moved away from `.less` on purpose. Like I mentioned at some point, Todoist will have to see how to deal with this while we cannot drop IE support. Probably by not using the components off of this repo for a while. I know it sucks, but I did not want our implementation here to be hindered by having to support IE11. Also, CSS custom properties allow apps using these components to customize the variables at the root and the design system will pick it up.

Reply:
No objections from my side, but this will probably mean extra work for you to keep reactist updated and to avoid the two implementations from diverging. Would you be ok with that?

Reply:
Yes. It would mean extra work either way (I already did the work here and taking it all back to .less would mean more work, plus having to import a bunch of .less variables all over the place is a big overhead as well. Not to mention that we'd need to write the CSS for dark mode twice.